### DISCUSSION TITLE
Support for accents

### DISCUSSION THREAD

**Discussion by ChenghaoMou** [srid/emanote#120] [Q&A] [answered]
This is the current output when emanote encounters a file called `el básquetbol.md`

```
emanote: /data/output.docker/Literature Notes/Spanish/el bsquetbol.html: openBinaryFile: invalid argument (invalid character)
```

Is it possible to escape the file name to something like `el%20b%C3%A1squetbol`?

**Comment by srid** [comment]
I do not see this (non-ascii letters in filename are already supported). Can you file a bug report with full repro steps?

**Reply by srid** [reply]
Okay I see it her: https://github.com/ChenghaoMou/chenghaomou.github.io/runs/3463022573#step:3:590

Weird ...

**Reply by srid** [reply]
It might be something to do with the Docker image (its lang encoding?), because I can run `emanote gen` successfully on a local computer using your repo.

**Comment by ChenghaoMou** [comment, marked as answer]
@srid 
Adding this solves the issue
```shell
docker run -e LANG=C.UTF-8 -e LC_ALL=C.UTF-8 ...
```

Thanks for the insights and quick response!
